python/lsst/ts/audiotrigger/read_serial_temp_scanner.py

```diff
@@ -0,0 +1,351 @@
+# This file is part of ts_audiotrigger.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Software.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["SerialTemperatureScanner"]
+
+import asyncio
+import functools
+import logging
+from collections import OrderedDict
+
+import pigpio
+import serial
+from lsst.ts import tcpip
+
+from .enums import Fan
+from .mocks import MockPio, MockThermalSensor
+
+
+class SerialTemperatureScanner(tcpip.OneClientServer):
+    """This is the class that implements the Serial Temperature Scanner script.
+
+    Script to readout temperature sensors from serial device for laser
+    The script is currently setup to write to a file every time a measurement
+    is made so that the file can be accessed while the script is running
```

Comment:
Please provide a correct description of what this class does. Remove all references to `script` and `write to file`.